### PR TITLE
Arrow: move arrow support from frontend to backend only

### PR DESCRIPTION
**Pull request by ryantxu**
This removes arrow from the frontend package, and supports it only via backend requests.

TODO:
- [x]  AlertingNG seems to use base64 encoding directly..  can we investigate that?
- [x]  update test snapshots

### CODE REVIEW

File: public/app/features/alerting/state/actions.ts [hunk 1]
```diff
@@ -233,8 +226,7 @@ async function buildAlertDefinition(state: AlertDefinitionState) {
 
 function handleBase64Response(frames: string[]) {
   const dataFrames = frames.map((instance) => {
-    const table = base64StringToArrowTable(instance);
-    return arrowTableToDataFrame(table);
+    return new MutableDataFrame(); // ????????????????????
```
[the remark this answers is not in the document]
Comment (torkelo) [reply public/app/features/alerting/state/actions.ts:229]: @ryantxu peter is out sick for next couple of weeks. 

Think this is used for the alert eval endpoints that return dataframes: 
https://github.com/grafana/grafana/blob/master/pkg/services/ngalert/api/api.go#L134
and
https://github.com/grafana/grafana/blob/master/pkg/services/ngalert/api/api.go#L169

[the remark this answers is not in the document]
Comment (ryantxu) [reply public/app/features/alerting/state/actions.ts:229]: any idea how to test/exercise this?

[the remark this answers is not in the document]
Comment (ryantxu) [reply public/app/features/alerting/state/actions.ts]: I pulled in the changes from #32580 here -- I think we can move ahead with this to simplify babel updates, and come back to fix alertng if there are issues

File: packages/grafana-runtime/src/utils/queryResponse.test.ts [hunk 2]
```diff
@@ -1,37 +1,79 @@
...
+        frames: [
```
Comment (dprokop) [inline packages/grafana-runtime/src/utils/queryResponse.test.ts:9]: Now I can finally read this without putting my glasses on ;)

**Review by dprokop — COMMENTED** [review]
Went through the current code and it looks good!

File: packages/grafana-runtime/src/utils/queryResponse.test.ts (hunk 2 above)
Comment (leeoniya) [reply packages/grafana-runtime/src/utils/queryResponse.test.ts:9]: hats off to you if all you need to read base64-encoded binary is glasses

**Review by leeoniya — APPROVED** [review]

**Comment by joshhunt** [comment]
@ryantxu I'm wondering if we should flag this as a breaking change for the 8.0 release, if this will break any plugins using the old tsdb endpoint which is still arrow encoded?

**Pull request merged**